src/DBConnection.hpp

```diff
@@ -4,12 +4,12 @@
 #include <utility>
 
 struct Info {
-    Info() = default;
+    Info() : server_version{""}, protocol_version(0) {};
```

Comment:
For whole file comment as above

Reply:
@malirod , there is a changes in constructor, therefore no repeats

Reply:
And why did you prefer such ctor instead of default values?

Reply:
@malirod , because of parametrized ctor existence